Directory.Build.targets

```diff
@@ -39,6 +39,16 @@
 		<Output TaskParameter="DestinationFiles" ItemName="FileWrites"/>
 		</Copy>
   </Target>
+
+  <Target Name="CopyDnnOnnxModels"
```

Comment:
Is there a reason to use a target rather than Content Items?

Reply:
Maybe make it work like this: https://github.com/dotnet/machinelearning/blob/b927a0f20009e0a271a9c74f086bb8290aa94bdb/src/Directory.Build.targets#L9